cmd/jet/main.go

```diff
@@ -344,3 +361,16 @@ func shouldSkipEnum(enum metadata.Enum, filter templateFilter) bool {
 
 	return !strslice.Contains(filter.names, strings.ToLower(enum.Name))
 }
+
+func createModelTags(columnMetaData metadata.Column) []string {
+	var tags []string
+	switch modelJsonTag {
+	case "snake-case":
+		tags = append(tags, fmt.Sprintf(`json:"%s"`, columnMetaData.Name))
```

Comment:
If a user chooses  `snake-case` then database names are probably in `camel-case`. 
`SnakeToCamel` function is originally copied from this library - https://github.com/serenize/snaker to avoid including the whole library. We can do the same for `CamelToSnake` function.